service/src/main/java/com/epam/esm/service/impl/UserServiceImpl.java

```diff
@@ -40,8 +56,49 @@ public UserDto findUserByIdService(Long id) {
         return userServiceMapper.convertUserToDto(user);
     }
 
+    @Transactional
     @Override
-    public Long count(){
```

Comment:
I saw you declared this method in the interface. If it is not necessary - delete it

Reply:
Its for pagination purposes